url.bs

```diff
@@ -1031,13 +1368,14 @@ actually doing that with the editors of this document first.
          <li>
           <p>If <var>ipv4Piece</var> is null, then set <var>ipv4Piece</var> to <var>number</var>.
 
-          <p>Otherwise, if <var>ipv4Piece</var> is 0, <a>validation error</a>, return failure.
+          <p>Otherwise, if <var>ipv4Piece</var> is 0, <a>IPv4-in-IPv6-invalid-first-part</a>
```

Comment:
This name is also wrong. The error here is the leading zero. This can happen in any part, not just the first.

Reply:
Good catch. Perhaps changing the name to something like "IPv4-in-IPv6-leading-zero" or "leading-zero-in-IPv4-part-in-IPv6"